chipseq_workflows/ChIP-seq_Peak_Calling_-_Mouse_(mm9).ga

```diff
@@ -0,0 +1,709 @@
+            "errors": null, 
+            "id": 1, 
+            "input_connections": {
+                "input_collection": {
+                    "id": 0, 
+                    "output_name": "output"
+                }
+            }, 
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool Split Paired Collection", 
+                    "name": "input_collection"
+                }
+            ], 
+            "label": null, 
+            "name": "Split Paired Collection", 
+            "outputs": [
+                {
+                    "name": "forward_element", 
+                    "type": "data"
+                }, 
+                {
+                    "name": "reverse_element", 
+                    "type": "data"
+                }
+            ], 
+            "position": {
+                "left": 474.5, 
+                "top": 376.5
+            }, 
+            "post_job_actions": {
+                "ChangeDatatypeActionforward_element": {
+                    "action_arguments": {
+                        "newtype": "fastqsanger"
+                    }, 
+                    "action_type": "ChangeDatatypeAction", 
+                    "output_name": "forward_element"
+                }, 
+                "ChangeDatatypeActionreverse_element": {
+                    "action_arguments": {
+                        "newtype": "fastqsanger"
+                    }, 
+                    "action_type": "ChangeDatatypeAction", 
+                    "output_name": "reverse_element"
+                }, 
+                "HideDatasetActionforward_element": {
+                    "action_arguments": {}, 
+                    "action_type": "HideDatasetAction", 
+                    "output_name": "forward_element"
+                }, 
+                "HideDatasetActionreverse_element": {
+                    "action_arguments": {}, 
+                    "action_type": "HideDatasetAction", 
+                    "output_name": "reverse_element"
+                }
+            }, 
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/refinery-platform/split_paired_collection/split_paired_collection/0.2", 
+            "tool_shed_repository": {
+                "changeset_revision": "3a60513b51a2", 
+                "name": "split_paired_collection", 
+                "owner": "refinery-platform", 
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            }, 
+            "tool_state": "{\"__page__\": 0, \"__rerun_remap_job_id__\": null, \"input_collection\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\"}", 
+            "tool_version": "0.2", 
+            "type": "tool", 
+            "uuid": "b9706229-8a52-47cc-bb8d-28d8103caae6", 
+            "workflow_outputs": []
+        }, 
+        "2": {
+            "annotation": "", 
+            "content_id": "toolshed.g2.bx.psu.edu/repos/kevyin/fastq_groomer_parallel/fastq_groomer_parallel/0.3.0", 
```

Comment:
Version 0.3.0 of `fastq_groomer_parallel` does not match version 0.3.1 for the rest of existing workflows (e.g.: https://github.com/stemcellcommons/scc-galaxy-tools/blob/dc4a5924dc98d22c14eac246506d13dfc13c3c6d/chipseq_workflows/ChIP-seq_Peak_Calling_-_Mouse.ga#L102).